FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1453,4 +1453,81 @@ public function testCategoryNamesWithSlashesAreExportedCorrectly($articleConfigu
             $cat_urls
         );
     }
+
+    public function productsWithPseudoSalesProvider()
+    {
+        return [
+            'Pseudo sale value of 0 is not exported' => [
+                'pseudoSales' => 0,
+                'expectedSalesFrequency' => 1
+            ],
+            'Pseudo sale value of 10 is exported' => [
+                'pseudoSales' => 10,
+                'expectedSalesFrequency' => 10
+            ]
+        ];
+    }
+
+    /**
+     * @dataProvider productsWithPseudoSalesProvider
+     *
+     * @param int $pseudoSales
+     * @param int $expected
+     *
+     * @throws Exception
```

Comment:
We do not really care about any `@throws` tags in tests, so this can be removed.